modules/metrics/src/test/java/org.motechproject.metrics/ut/config/MetricsConfigFacadeTest.java

```diff
@@ -0,0 +1,142 @@
+package org.motechproject.metrics.ut.config;
```

Comment:
It is kinda an unwritten rule, but we keep the unit tests in the same package as the tested class.